testgrid/config/config.yaml

```diff
@@ -845,7 +845,7 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gce-enormous-teardown
 - name: kubernetes-e2e-gke-gci-test-release-1.2
   gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gke-gci-test-release-1.2
-- name: testgrid-config-upload
+- name: maintainance-ci-testgrid-config-upload
```

Comment:
meh... I was referencing maintenance-ci-clean-project... I'll keep the name then